src/Xenial.Tasty/Reporters/SummaryReporter.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xenial.Delicious.Metadata;
+using Xenial.Delicious.Utils;
+
+namespace Xenial.Delicious.Reporters
+{
+    public static class SummaryReporter
```

Comment:
I'm not sure about this type. The pipeline only pumps trough collected tests to the original summary reporters anyway.

Reply:
So it would be something like -> AsyncTestReporter -> AsyncSummaryCollectors? -> AsyncSummaryReporters? in the pipeline?